doc/source/ray-air/examples/gptj_deepspeed_fine_tuning.ipynb

```diff
@@ -412,23 +400,24 @@
     "    ret[\"labels\"] = ret[\"input_ids\"].copy()\n",
     "    return dict(ret)\n",
     "\n",
-    "\n",
-    "splitter = BatchMapper(split_text, batch_format=\"pandas\")\n",
-    "tokenizer = BatchMapper(tokenize, batch_format=\"pandas\")"
+    "processed_datasets = {\n",
+    "    key: ds.map_batches(split_text, batch_format=\"pandas\").map_batches(tokenize, batch_format=\"pandas\")\n",
+    "    for key, ds in ray_datasets.items()\n",
+    "}\n",
+    "processed_datasets"
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Fine-tuning the model with Ray AIR <a name=\"train\"></a>\n",
+    "### Fine-tuning the model with Ray Train <a name=\"train\"></a>\n",
     "\n",
-    "We can now configure Ray AIR's {class}`~ray.train.huggingface.TransformersTrainer` to perform distributed fine-tuning of the model. In order to do that, we specify a `trainer_init_per_worker` function, which creates a 🤗 Transformers `Trainer` that will be distributed by Ray using Distributed Data Parallelism (using PyTorch Distributed backend internally). This means that each worker will have its own copy of the model, but operate on different data, At the end of each step, all the workers will sync gradients.\n",
+    "We can now configure Ray Train's {class}`~ray.train.torch.TorchTrainer` to perform distributed fine-tuning of the model. In order to do that, we specify a `train_loop_per_worker` function, which defines all the training logics that will be distributed by Ray using Distributed Data Parallelism (using PyTorch Distributed backend internally). This means that each worker will have its own copy of the model, but operate on different data, At the end of each step, all the workers will sync gradients.\n",
     "\n",
     "Because GPT-J is a relatively large model, it may not be possible to fit it on smaller GPU types (<=16 GB GRAM). To deal with that issue, we can use [DeepSpeed](https://github.com/microsoft/DeepSpeed), a library to optimize the training process and allow us to (among other things) offload and partition optimizer and parameter states, reducing GRAM usage. Furthermore, DeepSpeed ZeRO Stage 3 allows us to load large models without running out of memory.\n",
```

Comment:
```suggestion
    "Because GPT-J is a relatively large model, it may not be possible to fit it on smaller GPU types (<=16 GB GRAM). To deal with that issue, this example uses [DeepSpeed](https://github.com/microsoft/DeepSpeed), a library to optimize the training process and to offload and partition optimizer and parameter states, reducing GRAM usage. Furthermore, DeepSpeed ZeRO Stage 3 can load large models without running out of memory.\n",
```